### PR TITLE
Remove requirement on six

### PR DESCRIPTION
**Pull request by kba**
No need to be backwards-compatible with Python 2.x

### CODE REVIEW

**Comment by mikegerber** [comment]
I gave it a quick test using the dinglehopper test suite (includes a OCR-D test) and a manual test. It seems to fix my issue!

File: ocrd_models/ocrd_models/ocrd_page_generateds.py [hunk 1]
```diff
@@ -1229,7 +1229,7 @@ def __hash__(self):
     @property
     def id(self):
         if hasattr(self, 'pcGtsId'):
-            return self.pcGtsId
+            return self.pcGtsId or ''
```
Comment (mikegerber) [inline ocrd_models/ocrd_models/ocrd_page_generateds.py:1232]: Is this an accidental change? It does not seem related to six?

Comment (mikegerber) [reply ocrd_models/ocrd_models/ocrd_page_generateds.py:1232]: Ah, this is the generated file and behaviour seemed to have changed slightly. The question is if this is wanted or not

Comment (kba) [reply ocrd_models/ocrd_models/ocrd_page_generateds.py:1232]: Yes, this is intended, a change proposed by @bertsky to https://github.com/OCR-D/core/blob/master/ocrd_models/ocrd_page_user_methods/id.py which I forgot to regenerate the file for.
